src/Calendar.js

```diff
@@ -92,6 +93,19 @@ class Calendar extends React.Component {
     */
    viewAliases: PropTypes.object,
 
+   /**
+    * An array of physician availability objects to display on the calendar
```

Comment:
I'd remove the word "physician". It's making an assumption about who's using the calendar, and this isn't EMR code. It's a general calendar component.